addons/stock/tests/test_product.py

```diff
@@ -276,3 +276,103 @@ def test_search_product_template(self):
         res = self.env['product.template'].name_search(name='123', operator='not ilike')
         res_ids = [r[0] for r in res]
         self.assertNotIn(template.id, res_ids)
+
+    def test_qty_available_multi_warehouse_location_context(self):
+        """
+        Test computed quantities of a product with multiple warehouses and locations in context
+        """
+        warehouse1 = self.env["stock.warehouse"].create({"name": "Primary Warehouse", "code": "PWH",
+                                                         "delivery_steps": "pick_ship"})
```

Comment:
```suggestion
        warehouse1 = self.env["stock.warehouse"].create({
            "name": "Primary Warehouse", 
            "code": "PWH",
            "delivery_steps": "pick_ship",
        })
```
Can you also reformat all next `create` ?